### PR TITLE
Loader

### PR DESCRIPTION
**Pull request by sidra2022**

### CODE REVIEW

**Review by AngeCast — CHANGES_REQUESTED** [review]
Got some comments here. Have a look when you can.

File: packages/stencil/src/components/datacom-loader/datacom-loader.tsx
Comment (sidra2022) [reply packages/stencil/src/components/datacom-loader/datacom-loader.tsx]: Thanks for the suggestion, I will do it.

File: packages/stencil/src/components/datacom-loader/datacom-loader.tsx
Comment (sidra2022) [reply packages/stencil/src/components/datacom-loader/datacom-loader.tsx]: Done.

File: packages/stencil/src/components/datacom-loader/datacom-loader.tsx
Comment (sidra2022) [reply packages/stencil/src/components/datacom-loader/datacom-loader.tsx]: Thanks for the suggestion, I will do it.

**Review by AngeCast — APPROVED** [review]
Changes seem fine. Will merge this AFTER the latest dev work has been merged into the main branch.

File: apps/storybook/src/stories/datacom-loader-single.stories.tsx
Comment (AngeCast) [inline apps/storybook/src/stories/datacom-loader-single.stories.tsx:3]: DatacomButton import is not used. Please remove.

File: apps/storybook/src/stories/datacom-loader.stories.tsx
Comment (AngeCast) [inline apps/storybook/src/stories/datacom-loader.stories.tsx:38]: Seeing error linting on line 68 complaining about your loadingStatus state not having the correct type. Maybe you can set the type on the useState hook to utilize the `LoadingStatusType`?

File: apps/storybook/src/stories/datacom-loader.stories.tsx
Comment (AngeCast) [inline apps/storybook/src/stories/datacom-loader.stories.tsx:28]: This template isn't being used. Either remove it or use it in one of the stories below.

File: apps/storybook/src/stories/datacom-loader.stories.tsx
Comment (AngeCast) [inline apps/storybook/src/stories/datacom-loader.stories.tsx:32]: args parameter has an error complaining about having an 'any' type (i.e. it's not typed at all.) You should give this a type of the Loader props plus the label.

Have a look at the ContentTag story file (line 43) for an idea of how that can be done.

**Review by AngeCast — CHANGES_REQUESTED** [review]

File: apps/storybook/src/stories/datacom-loader.stories.tsx
Comment (sidra2022) [reply apps/storybook/src/stories/datacom-loader.stories.tsx]: It still shows a typed error even though the assigned type.

**Review by AngeCast — APPROVED** [review]

**Review by AngeCast — APPROVED** [review]

**Pull request merged**
